src/community/ogcapi/ogcapi-changeset/src/main/java/org/geoserver/api/changeset/TileIterator.java

```diff
@@ -103,7 +101,7 @@ private boolean nextRow() {
                             long[] coverage = ss.getCoverage(z);
                             return this.y >= coverage[1] && y <= coverage[3];
                         })
-                .sorted((ss1, ss2) -> Long.signum(ss1.getCoverage(z)[0] - ss1.getCoverage(z)[0]))
+                .sorted((ss1, ss2) -> Long.signum(ss1.getCoverage(z)[0] - ss2.getCoverage(z)[0]))
```

Comment:
This is a functional change. I don't understand the code that well, but I'm guessing this is what was intended.

Reply:
Agreed